Public/Src/Cache/ContentStore/Library/Sessions/ContentSessionBase.cs

```diff
@@ -211,10 +211,10 @@ protected ContentSessionBase(string name, CounterTracker counterTracker = null)
                 operationContext => operationContext.PerformOperationAsync(
                     Tracer,
                     () => PutFileCoreAsync(operationContext, hashType, path, realizationMode, urgencyHint, _counters[ContentSessionBaseCounters.PutFileRetries]),
-                    extraStartMessage: $"({path},{realizationMode},{hashType}) trusted=false",

```

Comment:
>hashType [](start = 68, length = 8)

Do we want to carry this along?

Reply:
I think we should keep path. Realization mode I'm not so sure of, but the whole purpose of this is to keep all the current information, so I think we could remove unnecessary info on a later PR.